R/exclude-potential-pcos.R

```diff
@@ -0,0 +1,54 @@
+#' Exclude metformin purchases potentially for the treatment of polycystic ovary syndrome.
+#'
+#' Takes the output from `include_gld_purchases()` and `bef` (information on sex and date
```

Comment:
```suggestion
#' Takes the output from [include_gld_purchases()] and `bef` (information on sex and date
```

Using `[]` is a way to autolink to these functions. https://roxygen2.r-lib.org/articles/markdown.html#links